painttiming.bs

```diff
@@ -62,9 +87,34 @@ Formally, we consider the user agent to have "rendered" a document when it has p
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the user agent is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
+<dfn>Element or pseudo-element</dfn>: A [=document-sourced element=] or a [=generated content pseudo-element=].
+<dfn>Contentful</dfn>: An [=element or pseudo-element=] counts as contentful when one or more of the following apply:
+* The element has a [=text node=] child, with at least one character, excluding [=document white space characters=].
```

Comment:
pseudo element never has a [child](https://dom.spec.whatwg.org/#concept-tree-child) since it's not a real node.